main.tf

```diff
@@ -98,6 +99,31 @@ module "nat_instances" {
   vpc_id                  = module.vpc.vpc_id
 }
 
+# ---------------------------------------------------------------------------------------------------------------------
+# Use of alterNAT instances instead of NAT instances to reduce costs on the basis of hourly usage.
+# ---------------------------------------------------------------------------------------------------------------------
+module "alternat_instances" {
+  count = local.create_alter_nat ? 1 : 0
+
+  source = "git::https://github.com/1debit/alternat.git//modules/terraform-aws-alternat?ref=v0.1.0"
+
+  alternat_image_uri = "531874807515.dkr.ecr.eu-central-1.amazonaws.com/alternat"
```

Comment:
Please use our public Docker account to publish the image, as this will incur cost on us. Or find an alternative solution.